### DISCUSSION TITLE
好有意思的网站，使我的英语大脑旋转

### DISCUSSION THREAD

**Discussion by wuwu0621** [share121/inter-knot#867] [其他]
![IMG_20240810_174715](https://github.com/user-attachments/assets/9526bbfe-5448-40f8-bf6f-bdf8e426a643)

**Comment by aaa00073** [comment]
我超，嘉心糖！

**Comment by Qiling-XwX** [comment]
![0BD4078C3800B0B832A5370CC8AAA19C](https://github.com/user-attachments/assets/ff4b443e-dd96-45f5-a1c0-ff34d097160f)

**Comment by Joker-JYH** [comment]
鼠标右键，找“翻译为 中文（简体）”
